Source/WebCore/editing/FrameSelection.cpp

```diff
@@ -1895,6 +1904,34 @@ void CaretBase::paintCaret(const Node& node, GraphicsContext& context, const Lay
 #endif
 }
 
+void FrameSelection::setCaretBlinkingSuspended(bool suspended)
+{
+    if (m_currentCaretAnimation)
+        m_currentCaretAnimation->setCaretBlinkingSuspended(suspended);
```

Comment:
This seems broken. We can't forget the fact caret blinking had been suspended
just because the animation wasn't scheduled at that point.